mlrun/utils/logger.py

```diff
@@ -352,11 +356,20 @@ def bind(self, **kw_args):
     def _update_bound_vars_and_log(
         self, level, message, *args, exc_info=None, **kw_args
     ):
+        request_id = request_id_var.get() or "N/A"
```

Comment:
can be left empty, I think
```suggestion
        request_id = request_id_var.get() or ""
```